### PR TITLE
Update box-connector-examples.adoc

### PR DESCRIPTION
**Pull request by patrik-patinak**

### CODE REVIEW

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 1]
```diff
@@ -250,6 +250,260 @@ output application/json
 </mule>
 ----
 
+== Anypoint Studio Example: Upload File
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Should just be: Example: Upload a File

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 2]
```diff
@@ -250,6 +250,260 @@ output application/json
...
+
+Since the Mule 4 version of Box connector doesn't provide upload file operation, this example demonstrates how to upload a file to Box.
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Perhaps we don't need to tell them that it doesn't provide a file upload operation. We could just say: This example demonstrates how to upload a file to Box.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 3]
```diff
@@ -250,6 +250,260 @@ output application/json
...
+
+. Create a new Mule Project in Anypoint Studio
++
+. Create a folder called 'input' under src/main/resources directory
++
+. Copy the file you want to upload to the 'input' directory
++
+. Configure the HTTP Listener by adding a new HTTP Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select HTTP Listener and click Ok. In the HTTP Listener screen configure the following parameters:
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Create needs to be in bold: *Create*, also *OK* should be bold.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 4]
```diff
@@ -250,6 +250,260 @@ output application/json
...
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name |HTTP Listener
+|Connector configuration | Select the HTTP Listener Configuration that had been created previously
+|Path |`/boxUploadFile`
+|===
++
+. Login to Box and Generate the Box developer token which is active for 60 minutes
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: a period at the end of the sentence.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 5]
```diff
@@ -250,6 +250,260 @@ output application/json
...
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name| Save developer token
+|Name |developerToken
+|Value | #[payload]
+|===
++
+. Configure the File connector by adding a new File Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select File Config and click Ok. In the File Config screen configure the following parameter:
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: File Global Element and Create need to be in bold. Also, OK.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 6]
```diff
@@ -250,6 +250,260 @@ output application/json
...
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|Display Name |Read File
+|Connector configuration | Select the File Configuration that had been created previously in Global Elements
+|File Path |`input/yourFileName`
+|===
++
+. Now you need to transform this payload to multipart/form-data type of request. You can also specify a folder to which will be file uploaded vby setting the ID of parent folder. Drag a DataWeave component next to the Read File component and use the script below.
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Delete 'Now you need to' as it is unecessary. Rewrite the following sentence: Specify a folder to be uploaded by setting the ID of the parent folder.

Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: rewrite to: use the following script.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 7]
```diff
@@ -250,6 +250,260 @@ output application/json
...
+            "filename": message.attributes.fileName
+        }
+      },
+      content : payload
+    }
+  }
+}
+----
++
+. Configure the HTTP Request by adding a new HTTP Request Global Element. Click on the Global Elements tab at the base of the canvas. In the Global Configuration Elements screen, click Create. In the Choose Global Type wizard, expand Connector Configuration and select HTTP Request Configuration and click Ok. In the HTTP Request Configuration screen configure the following parameters:
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Create and OK need to be bolded.

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 8]
```diff
@@ -250,6 +250,260 @@ output application/json
...
+|Parameter |Value
+|Display Name| Request - Box Upload File
+|Configuration| Select the HTTP request configuration that had been created previously in Global Elements
+|Method|POST
+|Path|
+|URL|https://upload.box.com/api/2.0/files/content
+|Body|payload
+|===
++
+. Now click we need to create Authorization header in the HTTP Request. Click on the Headers tab next to the Body in the HTTP request , click on the plus sign and configure the following parameter:
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: Rewrite to: 
Next create an Authorization header in the HTTP Request. Click the *Headers* tab next to the Body in the HTTP request, then click the plus sign and configure the following parameter:

File: modules/ROOT/pages/box/box-connector-examples.adoc [hunk 9]
```diff
@@ -250,6 +250,260 @@ output application/json
...
++
+[%header%autowidth.spread]
+|===
+|Name |Value
+|"Authorization"|vars.developerToken
+|===
++
+. Finally drag the DataWeave component to set the payload to display the result of the flow to the usert and use the script below.
```
Comment (mcherem) [inline modules/ROOT/pages/box/box-connector-examples.adoc]: following script.

**Review by mcherem — CHANGES_REQUESTED** [review]
I am not able to make these edits. Requesting that these be made before merging.

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 1 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: updated

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 2 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: updated

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 4 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: updated

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 3 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: updated

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 5 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: Updated text to bold

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 6 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: done

Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: done

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 7 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: Text is bold now

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 8 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: Rewrote

File: modules/ROOT/pages/box/box-connector-examples.adoc (hunk 9 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/box/box-connector-examples.adoc]: updated

**Review by mviswana — APPROVED** [review]

**Comment by mcherem** [comment]
Fixed per the following PR: Update Box Connector with technical changes #338

closing this one.

**Pull request closed without merging**